internal/driver/driver.go

```diff
@@ -80,6 +84,10 @@ func (s *Driver) HandleReadCommands(deviceName string, protocols map[string]mode
 	for i, req := range reqs {
 		s.lc.Debugf(fmt.Sprintf("Driver.HandleReadCommands(): protocols: %v resource: %v attributes: %v", protocols, req.DeviceResourceName, req.Attributes))
 
+		// Get the value type from device profile
+		valueType := req.Type
+		s.lc.Debugf(fmt.Sprintf("Driver.HandleReadCommands(): value type = %v", valueType))
```

Comment:
Remove use of embedded `fmt.Sprintf` here and other places it is used in log messages.
```suggestion
		s.lc.Debugf("Driver.HandleReadCommands(): value type = %v", valueType)
```

Reply:
Addressed.